docs/builtins/Operators/and-q.md

```diff
@@ -1,33 +1,38 @@
 ## and?
-Use `and?` to apply a logical AND operation to the results of applying a `value` to `a` and `b`, with short-circuiting.
+
+Use `and?` to apply a logical AND operation to the results of applying a specified `value` to application functions `oper1` and `oper2`, with short-circuit evaluation.
 
 ### Basic syntax
 
-To apply logical 'and' to the results of applying `value` to `a` and `b`, use the following syntax:
+To apply a logical AND operation to the results of applying a specified `value` to the functions `oper1` and `oper2`, use the following syntax:
 
-`(and? a b value)`
+```pact
+(and? oper1 oper2 value)
+```
 
 ### Arguments
 
 Use the following arguments to specify the functions and `value` for the `and?` operation.
 
 | Argument | Type | Description |
 | --- | --- | --- |
-| `a` | `x:<r> -> bool` | Specifies the first function to apply `value` to. |
-| `b` | `x:<r> -> bool` | Specifies the second function to apply `value` to. |
-| `value` | `<r>` | Specifies the value to apply to both `a` and `b` functions. |
+| `oper1` | function | Specifies the first function to apply the specified `value` to. The result of applying the specified value returns a boolean value. |
```

Comment:
I am unsure whether i like this. Maybe `function x:<r> -> bool`. 

The types simply aren't clear here.

Reply:
There are a few options we can pursue.
If you want to use something like `x:<r> -> bool` as a type, we need to create a documentation conventions section that clearly explains what this means to an uninitiated person. Part of the problem in using stuff like x, <x>, <r>, and <a>  is that I often have no idea what it is trying to convey. 
I would prefer that we add more descriptive information about the functions and types expected (somewhat like the Rust example, I think).  I realize using `function` here is probably too much of a punt, but in trying to test the examples of the functions, I found some of the type description just too cryptic to be useful.

The two most likely fixes in my mind would be:
- Doc conventions with a few paragraphs explaining what the hieroglyphic notations are meant to convey that can be applied to all cases where they are used. 
- A Functions (or something like that) subsection where it's applicable that explains what's expects in the functions that are arguments for other functions.

I don't think there are a ton of these. But I did stumble over several like the and? that were hard for me to figure out.